src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -1134,8 +1134,9 @@
                                                                     </DataTrigger>
                                                                     <DataTrigger Binding="{Binding Path=FileName, Converter={StaticResource PathToSaveStateConverter}}"
                                                                                  Value="Saved">
+                                                                        <!--TODO find out what is up with this binding did it ever do anything?-->
                                                                         <Setter Property="Header"
-                                                                                Value="{Binding FileName, Converter={StaticResource PathToFileNameConverter}}" />
```

Comment:
@QilongTang  this seems like an invalid binding to me - I'm not even sure what part of the UI this is for, any ideas? It looks like this code was last modified by Peter 3 years ago.

Reply:
I dont seem to understand this either

Reply:
@QilongTang I figured it out... some UI I didn't even know we had:
![screen shot 2018-01-04 at 1 04 53 pm](https://user-images.githubusercontent.com/508936/34577669-f187c0aa-f14f-11e7-8b3a-0ce1cea0fc01.png)

The 3 line hamburger control opens a list of tabs - and it uses these other bindings I think.